Makefile

```
@@ -17,17 +17,12 @@ image:
 		$(CURDIR)
 
-image-otel:
-	$(DOCKER) build \
-		-f Containerfile \
-		--build-arg FACT_VERSION=$(FACT_VERSION) \
-		--build-arg RUST_VERSION=$(RUST_VERSION) \
-		--build-arg CARGO_ARGS="--features otel" \
-		-t $(FACT_IMAGE_NAME)-otel \
-		$(CURDIR)
+image-otel: CARGO_ARGS = --features otel
+image-otel: image
```

checkmake (0.3.2)

[warning] Target "image-otel" defined multiple times (lines 24 and 25).

(uniquetargets)